pkg/secretless/entrypoint_test.go

```diff
@@ -0,0 +1,82 @@
+package secretless
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"regexp"
+	"testing"
+
+	"github.com/cyberark/secretless-broker/internal/pkg/plugin"
+)
+
+var testManager = &plugin.Manager{}
+
+func assertOutputMatches(t *testing.T, params *CLIParams, regexString string) {
+	compiledRegex := regexp.MustCompile(regexString)
+
+	// Swap our stdout with a special one for capture
+	oldStdout := os.Stdout
+	oldStderr := os.Stderr
+
+	stdoutReadPipe, stdoutWritePipe, err := os.Pipe()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	stderrReadPipe, stderrWritePipe, err := os.Pipe()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	os.Stdout = stdoutWritePipe
+	os.Stderr = stderrWritePipe
+
+	stdoutOutputChan := make(chan string)
+	stderrOutputChan := make(chan string)
+
+	// Copying the output in a separate goroutine otherwise printing can block
+	go func() {
+		var buf bytes.Buffer
+		io.Copy(&buf, stdoutReadPipe)
+		stdoutOutputChan <- buf.String()
+	}()
+
+	go func() {
+		var buf bytes.Buffer
+		io.Copy(&buf, stderrReadPipe)
+		stderrOutputChan <- buf.String()
+	}()
+
+	Start(params, testManager)
+
+	// Restore stdout
+	stdoutWritePipe.Close()
+	stderrWritePipe.Close()
+
+	os.Stdout = oldStdout
+	os.Stderr = oldStderr
+
+	stdoutOutput := <-stdoutOutputChan
+	stderrOutput := <-stderrOutputChan
```

Comment:
there’s no assertion on `stderrOutput` so i wonder if that’s at all necessary. i don’t think we use stderr anywhere else in the project